IbisLib/ibisapi.cpp

```diff
@@ -19,6 +19,7 @@
 
 const int IbisAPI::InvalidId = SceneManager::InvalidId;
 const QString IbisAPI::MINCToolsPathVarName = "MINCToolsDirectory";
+const QString IbisAPI::PlusServerExecutablePath = "PlusServerExecutablePath";
```

Comment:
IbisAPI shouldn't know anything about a specific plugin. This must be defined in the plugin itself.